server/controllers/conversation.js

```diff
@@ -25,31 +27,125 @@ exports.getUserConversations = asyncHandler(async (req, res) => {
 });
 
 // @route POST /users/conversation/:userId
+// create a conversation with another user
 exports.postUserConversation = asyncHandler(async (req, res) => {
   const userId = req.user.id;
   const otherUserId = req.params.userId;
 
+  const currentUser = await User.findById(userId);
+  const theOtherUser = await User.findById(otherUserId);
   const conversationExists = await Conversation.findOne({
     users: [otherUserId, userId].sort(),
   });
 
   if (conversationExists) {
-    return res.status(200).json({ id: conversationExists._id });
+    return res.status(200).json({ message: "Conversation already exists" });
```

Comment:
```suggestion
    return res.status(400).json({ message: "Conversation already exists" });
```
I realize this was already a 200 in the code before this PR, bbut probably better for it to be a 400